src/site/markdown/server/manage-webadmin.md

```diff
@@ -412,7 +412,7 @@ The kind of task scheduled depends on the action parameter. See below for detail
 This task is only available on top of Guice Cassandra products.
 
 ```
-curl -XPOST -H "Confirmation: true" /mailboxes?task=SolveInconsistencies
+curl -XPOST -H "I-KNOW-WHAT-I-M-DOING: ALL-SERVICES-ARE-OFFLINE" /mailboxes?task=SolveInconsistencies
```

Comment:
I would not provide a copy/paste example in the doc because as long as people can execute things without reading they will.